terraform/terraform.tfvars

```diff
@@ -0,0 +1,31 @@
+# Generics
```

Comment:
I think that this file should be completed by the customer with the values he wants.
Maybe we can keep if empty and show examples using comments.

For example:
```
region =  # Example: "us-east-1"
```

What do you think?

Reply:
tfvars is the file where the customer will add his own values for the variables defined in it. I think the use case is included in the current set-up.

Reply:
Ok, so in that case, I prefer to have it empty, so if the customer forgot to complete it, the script will through an error.